tests/evaluate_test.py

```diff
@@ -20,12 +20,13 @@
 import tempfile
 
 import bayesnf
-from bayesnf import dataset_config
-from bayesnf import evaluate
 import jax
 import pandas as pd
 import pytest
 
+import dataset_config
```

Comment:
can we leave the import as it was and avoid the symlinks?

Reply:
The `dataset_config` module is no longer in the `bayesnf` namespace; it has been moved to the `scripts/` directory which is not released with the library... Aside from a symlink, we could use `sys.path.insert('../scripts')` or some similar workaround. What do you suggest?